app/services/settings-v2/video.ts

```diff
@@ -5,78 +5,230 @@ import { mutation, StatefulService } from '../core/stateful-service';
 import * as obs from '../../../obs-api';
 import { SettingsManagerService } from 'services/settings-manager';
 
+/**
+ * Display Types
+ *
+ * Add display type options by adding the display name to the displays array
+ * and the context name to the context name map.
+ */
+const displays = ['default', 'horizontal', 'green'] as const;
```

Comment:
I didn't want to remove the default property just yet in production to prevent anything possible referencing it and breaking. But I can go ahead and do that.